### ISSUE TITLE
Include interesting harmonics in Ouroboros mode

### ISSUE THREAD

**Issue by nifgam** [qiemem/eurorack#37]
Thanks for the Stages firmware - absolutely brilliant! The advanced segment generator mode, especially, I will be using a lot.
One thing I'd like to see, if you ever develop this further, is a slight change to harmonic oscillator. Currently, all the choices for partials are octaves of the fundamental or fifth apart from one, the 5x (major third). If there are to be 9 choices, I would have liked to have seen the 9 harmonics 2x, ... 10x, or at least an inclusion of the 7x harmonic which gives a nice flavour to the sound, IMO.
I only mention this as I imagine it is an easy change of code. 

(I could probably do it myself if I knew how to compile it / make a wav file. I am familiar with C++ and run Linux. If there is a howto page somewhere for this that you could point me to, that would be great - I couldn't find one. Otherwise, no worries.)

Thanks again!

**Comment by qiemem** [comment]
Glad you're enjoying the firmware! While it is an easy change, I wouldn't want to change the default for everyone. I was actually working on a UI for changing harmonic range of each segment (#21), but the (naive) approach I took incurred too big of a performance hit, and audio got choppy as a result. I hope to revisit at some point, but may not have time for a bit.

Luckily, it's also quite easy to get setup building firmware! Émilie has graciously setup a vagrant environment to make building very easy. Check out: https://github.com/pichenettes/mutable-dev-environment. Just pull this repo instead of pichenettes/eurorack and you'll be good to go. This is the line you'll then want to change: https://github.com/qiemem/eurorack/blob/eff0af4c42186e86e4673d1a64059780a13c9fba/stages/stages.cc#L212

Closing issue because the implementation will be the same as #21
